src/callables.c

```diff
@@ -19,7 +19,7 @@ SEXP exp_vec_cast(SEXP x, SEXP to) {
 }
 
 SEXP exp_vec_chop(SEXP x, SEXP indices) {
-  return vec_chop(x, indices);
+  return vec_chop_unsafe(x, indices, r_null);
 }
```

Comment:
Exposed experimental `exp_vec_chop()` callable does not gain the `sizes` argument